src/anomalib/data/base/dataset.py

```diff
@@ -172,11 +172,10 @@ def _setup(self) -> DataFrame:
             mask_path (if task == "segmentation"): path to the ground truth masks (for the anomalous images only).
 
         Example:
-        -------
-        |---|-------------------|-----------|-------------|------------------|-------|
-        |   | image_path        | label     | label_index | mask_path        | split |
-        |---|-------------------|-----------|-------------|------------------|-------|
-        | 0 | path/to/image.png | anomalous | 1           | path/to/mask.png | train |
-        |---|-------------------|-----------|-------------|------------------|-------|
+            |---|-------------------|-----------|-------------|------------------|-------|
+            |   | image_path        | label     | label_index | mask_path        | split |
+            |---|-------------------|-----------|-------------|------------------|-------|
+            | 0 | path/to/image.png | anomalous | 1           | path/to/mask.png | train |
+            |---|-------------------|-----------|-------------|------------------|-------|
```

Comment:
I've noticed that this table is not properly shown on the documentation. We need to make this a bit html friendly. Of course, in another PR when we are working on the documentation :)